Mozilla_VPN_ToS/en-US.md

```diff
@@ -0,0 +1,80 @@
+# Mozilla VPN Terms of Service
+
+Version 3.0, effective July 15, 2020
```

Comment:
@litkaemily I just realize the page title and the date format are different from the Privacy Notice docs. Is this intentional or should they be consistent for all legal docs?